aiidalab/app.py

```diff
@@ -727,7 +728,7 @@ def __init__(self, name, app_data, aiidalab_apps_path, watch=True):
         except KeyError:
             raise ValueError(f"Did not find categories in {self.name} metadata")
 
-        self.is_installed = self._app.is_installed
+        self.is_installed = self._app.is_installed()
```

Comment:
This is a bad practice that we mixed `is_installed()` as a function in `_AiidaLabApp` and as a property like traitlets in `AiidaLabApp` class. However, this function also called in `aiidalab-home` and `aiidalab-widgets-base`, I don't know how to make it backward compatible and add a deprecation message. Is that possible to keep it in `_AiidalabApp` class as a function call and as a property with the same name? 

I tried:
```python
    @property
    def is_installed(self) -> bool:
        """The app is installed if the corresponding folder is present."""
        return self.path.exists()
    
    def is_installed(self) -> bool:
        """deprecated alias for is_installed"""
        warning.warn(
            "The is_installed method is deprecated, use the is_installed property instead.",
            DeprecationWarning,
        )
        return self.is_installed
```
But the `is_installed` is still regarded as a function call first.

@sphuber I think there were similar change in aiida-core, how you managed those changes?

Reply:
> I think there were similar change in aiida-core, how you managed those changes?

I don't think so. We may have had a similar problem, but doubt we found a smooth deprecation pathway. I can't think of a way anyway right now.

Reply:
Yeah, let's not do any further changes that are not strictly necessary. We should aim to get this all working so we can release a new version of both Docker stacks soon.

Reply:
Sorry for the confusion, I was talking about `installed_version` method. Please see the comment reply below.